week03_model_free/homework.ipynb

```diff
@@ -758,6 +758,59 @@
     "Also, don't forget to optimize ```learning_rate```, ```discount``` and ```epsilon``` params of model, this may also help to solve this env."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# The following commands install the environment \"Pacman\" requirements\n",
+    "# in Collab, the course docker container, Ubuntu and other OS that use the package manager \"APT-GET\"\n",
+    "#!apt-get update\n",
+    "#!apt-get install -y python-tk"
```

Comment:
1. Ok
2. I got an exeception that force me to install python-tk. It should be installed or "run_*" scripts should be modificated in purpose of turning visualization by default off.
3. I'm not sure that sudo works in Colab or docker but I'm sure that the course student are able to read errors and add sudo if it is necessary.
4. Ok about naming. Lets suppose that it can be some utilities that copy the name and the behaivor of the "apt" on other OS.

Reply:
1 and 4 are done